src/components/selection-controls/Checkbox.js

```diff
@@ -12,7 +12,11 @@ export default {
   },
 
   props: {
-    indeterminate: Boolean
+    indeterminate: Boolean,
+    ripple: {
+      type: [Boolean, Object],
+      default: true
```

Comment:
`default: () => ({ center: true })` recommended @johnleider  in https://github.com/vuetifyjs/vuetify/pull/1182